<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>430</y>
     <width>361</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QCheckBox" name="ShowRowHeader">
      <property name="text">
       <string>显示行表头</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ShowListHeader">
      <property name="text">
       <string>显示列表头</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="AlternatingDisplay">
      <property name="text">
       <string>交替显示背景色</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_13">
    <item>
     <widget class="QLineEdit" name="KeyWords"/>
    </item>
    <item>
     <widget class="QPushButton" name="Search">
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>82</y>
     <width>361</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>教材编号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ISBNEditor"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>教材名称</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="NameEditor"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>教材数量</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="CountEditor"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>教材价格</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="PriceEditor"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>所属专业</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="SubjectEditor"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>作者</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="AuthorEditor"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>出版社名称</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="PublisherEditor"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>备注信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="RemarksEditor"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>保存时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="TimeShower"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="Append">
        <property name="text">
         <string>添加一行</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Insert">
        <property name="text">
         <string>插入一行</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Delete">
        <property name="text">
         <string>删除当前行</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Edit">
        <property name="text">
         <string>修改当前行</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="TablePosition">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>50</y>
     <width>123</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>当前位置：1行1列</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>641</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTableWidget" name="TextBooksTable"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="Import">
          <property name="text">
           <string>从表格导入数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Export">
          <property name="text">
           <string>导出数据至表格</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="Clear">
      <property name="text">
       <string>清空数据</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
